sigstore/_verify.py

```diff
@@ -236,27 +246,91 @@ def verify(
 
         logger.debug("Successfully verified signature...")
 
-        # Retrieve the relevant Rekor entry to verify the inclusion proof and SET
-        entry = self._rekor.log.entries.retrieve.post(
-            signature.decode(),
-            sha256_artifact_hash,
-            base64.b64encode(certificate).decode(),
-        )
-        if entry is None:
-            return RekorEntryMissing(
-                signature=signature.decode(), sha256_artifact_hash=sha256_artifact_hash
+        entry: Optional[RekorEntry]
+        if offline_rekor_entry is not None:
+            # TODO(ww): This should all go in a separate API, probably under the
```

Comment:
NTS: I put this here just so people can review it, but I'll factor it into a separate API as part of this PR.

Reply:
Actually, I'm going to postpone this to #250, since that refactor will include this functionality.